kernelci/cli/base.py

```diff
@@ -520,9 +525,24 @@ def __init__(self, path, command, cli_args, section=None):
                 if os.path.exists(default_path):
                     path = default_path
                     break
-        self._settings = configparser.ConfigParser()
+
+        if path and path.endswith('.conf'):
+            self._deprecated_settings = True
+
+        if self._deprecated_settings:
+            print("Warning: user settings file format '.conf' will soon be \
+deprecated. Please use '.toml' file format and provide 'kernelci.toml' file \
+instead.")
```

Comment:
```suggestion
instead.", file=sys.stderr)
```
The stdout data is used as input for other commands, so printing arbitrary messages on stdout can break things.

Reply:
Fixed it. Thanks.